balboa-http/src/main/scala/com/socrata/balboa/server/MetricsServlet.scala

```diff
@@ -29,15 +29,6 @@ class MetricsServlet extends JacksonJsonServlet
   val CombineKey = "combine"
   val FieldKey = "field"
 
-  val seriesMeter = com.yammer.metrics.Metrics.newTimer(
```

Comment:
@jacobsi It doesn't look like these were being used or displayed anywhere, so I've just removed them. It might be worth investigating a better long-term replacement along the lines of EN-679.